gpfit/plot_fit.py

```diff
@@ -55,4 +58,4 @@ def plot_fit_1d(u, w, K=1, fitclass='MA', plotspace='log'):
     n = 51
     u = np.logspace(0, np.log10(3), n)
     w = (u**2+3) / (u+1)**2
-    f = plot_fit_1d(u, w, K=2, fitclass='SMA', plotspace='linear')
+    f = plot_fit_1d(u, w, K=2, fitclass='MA', plotspace="linear")
```

Comment:
I'm super guilt of this, but we should really decide whether we use single quotes or doubles quotes. At the very least, we should be consistent within the same line.